windows/visual-studio/packer.yaml

```diff
@@ -23,8 +23,8 @@ builders:
       filters:
         virtualization-type: "hvm"
         root-device-type: "ebs"
-        name: "Windows_Server-2019-English-Full-ContainersLatest-*"
-      owners: ["801119661308"]
+        name: "Windows_Server-2019-English-Full-EKS_Optimized-*"
+      owners: ["687402702948"]
```

Comment:
Following is the command searched for the image and owner id 
```
aws ec2 describe-images \ 
    --owners amazon \
    --filters "Name=name,Values=Windows_Server-2019-English-Full-*" \
    --query "sort_by(Images,&Name)[].[Name,ImageId,OwnerId]"
[
    [
        "Windows_Server-2019-English-Full-Base-2023.07.12",
        "ami-08194a5af62132f6e",
        "801119661308"
    ],
...

    [
        "Windows_Server-2019-English-Full-ECS_Optimized-2023.07.11",
        "ami-0c90198f9a79fd57a",
        "196427335492"
    ],
...
    [
        "Windows_Server-2019-English-Full-EKS_Optimized-1.23-2023.07.11",
        "ami-0a81923768d17bb75",
        "687402702948"
    ],
...
```

Reply:
Note: The results returned would be different if a different region was set. I did not see the owner id "687402702948" until I explicitly searched for amis in "ap-southeast-2".